drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/main/java/org/drools/workbench/screens/dtablexls/backend/server/conversion/DecisionTableXLSToDecisionTableGuidedConverter.java

```diff
@@ -163,7 +167,7 @@ public ConversionResult convert( final Path path ) {
         ConversionResult result = new ConversionResult();
 
         //Check Asset is of the correct format
-        if ( !xlsDTableType.accept( path ) ) {
+        if ( !( xlsDTableType.accept( path ) || xlsxDTableType.accept( path ) ) ) {
             result.addMessage( "Source Asset is not an XLS Decision Table.",
```

Comment:
Just a minor thing, maybe also in the error message should be mentioned as XLS so XLSX extension.